src/DayColumn.js

```diff
@@ -84,6 +84,15 @@ class DayColumn extends React.Component {
     this.slotMetrics = this.slotMetrics.update(nextProps)
   }
 
+  componentDidUpdate(prevProps) {
+    if (prevProps.date.getTime() !== this.props.date.getTime()) {
+      if (this.props.isNow) {
+        this.positionTimeIndicator()
+        this.triggerTimeIndicatorUpdate()
```

Comment:
This is not useful to trigger the setTimeout again, the initial one is still running.

Reply:
I think it makes sense to trigger it here, the time has explicitly changed and you may not want to wait up to a minute for that to show